README.md

```diff
@@ -40,13 +40,13 @@ If you rather use node-like pipes, check the [eslint-eslines](https://github.com
 
 ## Config file
 
-`eslines` reads its configuration from a file named `.eslines.json` placed in the root of your git repository. Out of the box, it comes with three ways of post-processing an ESLint report - we call them *processors*: `downgrade-unmodified-lines`, `filter-parsing-errors`, `enforce`.
+`eslines` reads its configuration from a file named `.eslines.json` placed in the root of your git repository. Out of the box, it comes with four ways of post-processing an ESLint report - we call them *processors*: `downgrade-unmodified-lines`, `filter-parsing-errors`, `filter-when-format`, `enforce`.
 
 Example config:
 
 	{
 		"branches": {
-			"default": ["downgrade-unmodified-lines", "enforce"],
+			"default": ["downgrade-unmodified-lines", "filter-when-format", "enforce"],
 			"master": ["filter-parsing-errors"],
 			"my/topic-branch": ["filter-parsing-errors"]
```

Comment:
Let's use the `filter-when-format` for the `my/topic-branch` and use `downgrade-unmodified-lines` and `enforce` as `default`. I'd like to have those together to clearly explain the difference between `enforce` and not downgrading, but adding a third processor to the mix makes thing a bit more complicated to understand for a first-time user.